pkg/blob/nodeserver.go

```diff
@@ -260,7 +285,7 @@ func (d *Driver) NodeStageVolume(ctx context.Context, req *csi.NodeStageVolumeRe
 		return &csi.NodeStageVolumeResponse{}, nil
 	}
 
-	_, accountName, _, containerName, authEnv, err := d.GetAuthEnv(ctx, volumeID, protocol, attrib, secrets)
```

Comment:
since the existing blob fuse impl needs the key, are the key being exchanged with blobfuse in a different manner?

Reply:
GetAuthEnv returns the `authEnv` value  which contains environ variables passed to the blobfuse process. The key is in those env values. This code is around line 470 in pkg/blob/blob.go